tensorboard/components/tf_backend/test/backendTests.ts

```diff
@@ -170,6 +184,20 @@ describe('backend tests', () => {
         this.router = createRouter(base, /*demoMode=*/true);
       });
 
+      it('ignores leading slash in dataDir', () => {
+        const router = createRouter('/data/', /*demoMode=*/false);
```

Comment:
Shouldn't demoMode be true if these are tests for demoMode?  Not sure it's necessary to test separately though given that this isn't really critical when in demo mode.